src/main/java/icbm/classic/content/blocks/launcher/screen/TileLauncherScreen.java

```diff
@@ -86,6 +103,20 @@ public void update()
 
         if (isServer())
         {
+            // salvo logic
+            if (isFiring) {
+                if (fireBucket.size() == 0) {
+                    isFiring = false;
+                    fireBucketSimulate = true;
+                    nextFireTick = 0;
+                }
+                // fire every 3 sec
+                nextFireTick++;
+                if (nextFireTick > 20*3) {
```

Comment:
Rather not hard code it to 3 seconds.

Reply:
Temporarily on 3 seconds until i do the ui input

Reply:
Would want a UI input first. As this might be unexpected behavior for players to work around.

Reply:
ui input will be done on this pr, just not doing it right now